src/definitions/modules/modal.js

```diff
@@ -138,6 +138,9 @@ $.fn.modal = function(parameters) {
         },
 
         destroy: function() {
+          if (module.observer) {
```

Comment:
`if (observer)` should do it alone, because this variable is already declared here https://github.com/Semantic-Org/Semantic-UI/blob/0a42dcdcb400d6fc275b04d6cb052a48efd68536/src/definitions/modules/modal.js#L79